src/Domain/DomainHelpers.hpp

```diff
@@ -178,7 +181,8 @@ size_t which_wedge_index(const ShellWedges& which_wedges);
  */
 std::vector<domain::CoordinateMaps::Wedge<3>> sph_wedge_coordinate_maps(
     double inner_radius, double outer_radius, double inner_sphericity,
-    double outer_sphericity, bool use_equiangular_map,
+    double outer_sphericity, double cube_half_length,
+    std::array<double, 3> focal_offset, bool use_equiangular_map,
```

Comment:
Combine these two parameters into `std::optional<std::pair<double, std::array<double, 3>>>`, move the argument after `use_equiangular_map`, and make it default to `std::nullopt`. That way you don't have to make up values when it doesn't make sense to specify these arguments. And there's less changes to existing function calls for this.